csharp/src/Ice/IRequestHandler.cs

```diff
@@ -31,7 +31,7 @@ internal interface IRequestHandler
         /// <param name="cancel">The cancellation token to cancel the sending of the request</param>
         /// <returns>A task if the request is a twoway request or null if it's a oneway request. The returned task can
```

Comment:
Unrelated to this PR , the returns comment must be updated

Reply:
I believe Benoit will be removing this interface.